contracts/src/TokenSpoke/interfaces/INativeTokenSpoke.sol

```diff
@@ -19,7 +19,12 @@ import {ITokenSpoke} from "./ITokenSpoke.sol";
  */
 interface INativeTokenSpoke is ITokenSpoke, INativeTokenBridge {
     /**
-     * @dev Emitted when reporting burned tx fees to hub chain.
+     * @notice Emitted when native tokens are burned for bridge transfers.
+     */
+    event BurnedForBridge(uint256 amount);
```

Comment:
I think the `BurnedForBridge` event added here purely duplicated information from the `TokensSent` event that would be emitted any time tokens are sent from a native token spoke instance right? If so, I don't think it's necessary to add.

It would also be a bit confusing why the event is emitted by `NativeTokenSpoke` instances, but not `ERC20TokenSpoke` instance when they burn tokens to bridge to other chains.

Reply:
I was mostly trying to cover the case when there's a change in balance of the bridge contract's native tokens with an event, and to have it easier to check for in unit tests that this burn was called. But do agree it's strange to just have in `NativeTokenSpoke` but not in `ERC20TokenSpoke`, so okay with removing.